distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1497,6 +1497,7 @@ type Table
        suffixing strategy.
     parse_to_columns : Text | Integer -> Text -> Case_Sensitivity -> Boolean -> Problem_Behavior -> Table
     parse_to_columns self column pattern="." case_sensitivity=Case_Sensitivity.Sensitive parse_values=True on_problems=Report_Error =
+        _ = [column, pattern, case_sensitivity, parse_values, on_problems]
```

Comment:
Also adding this in my PR 😅 
I think it would be cool to have the `--warnings-as-errors` option to enable it on CI. Shall we create a ticket?